### DISCUSSION TITLE
graphql/scalars/docs/scalars/object-id

### DISCUSSION THREAD

**Discussion by giscus** [Urigo/graphql-scalars#1779] [Docs Discussions]
# graphql/scalars/docs/scalars/object-id

GraphQL Scalars Documentation

https://the-guild.dev/graphql/scalars/docs/scalars/object-id

**Comment by hoffnung8493** [comment]
looks like it will return string once its validated using regex. Shouldnt it return an ObjectId instance?

**Reply by leandroruel** [reply]
yes
